test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,30 @@
 
 public class UtilsTest {
 
+   @Test
+   public void isAnyNull_null_returnsFalse() throws Exception {
```

Comment:
The `scenarioBeingTested` is not clear here. Should be named something other than just `null`. The format is `featureUnderTest_scenarioBeingTested_expectedOutcome`.